### ISSUE TITLE
Instructions on how to set up reminder emails?

### ISSUE THREAD

**Issue by zz-pony** [Khan/snippets#29]
I found cron.yaml which is supposed to send out reminder emails. But following the README.md won't set up reminder emails, I guess? So how to properly set up this feature? Thanks.

**Comment by zz-pony** [comment]
Thanks. I ran `gcloud app deploy cron.yaml --project "$MY_PROJECT_NAME"` and cron job was set up successfully. I could update README to include this line.

However, the free quota was not enough for email reminders (only 10 emails daily). Is it common practice to set up billing info to receive the free 100 emails quota per day?

**Comment by zz-pony** [comment]
Also, I believe I also set up email sender in the app following this page: https://cloud.google.com/appengine/docs/standard/python/mail/#who_can_send_mail. In my case, I simply used `noreply@mydomain.com`. I believe we should set default sender to this address as well?

**Comment by csilvers** [comment]
Yeah, the free quota is very limiting here for sure.  I think I set up billing so I could get 100 emails per day, which was enough for my needs.

> I believe we should set default sender to this address as well?

I think that this is something that would need to be configured on a per-install basis.  We could add support for that in the config page, or maybe do it some other way.  Would be glad to see a pull request.

**Comment by tedpower** [comment]
BTW if you're using vanilla App Engine, you'll want an email in this form: <noreply@[your-app-name].appspotmail.com>
